src/main/java/model/config/Config.java

```diff
@@ -0,0 +1,19 @@
+package model.config;
+
+import model.JSONable;
+import model.data.ConfigID;
+
+/**
+ * 
+ * Base class for all model classes that require JSON serialization.
+ * 
+ * @author jonathanhasenburg
+ *
+ */
+public abstract class Config implements JSONable {
+	
+	abstract ConfigID getID();
```

Comment:
We might need to add an @JSONIgnore so that it does not mess up JSONs generated by subclasses